src/content/LandingPage/_landing-page.scss

```diff
@@ -0,0 +1,54 @@
+@import 'carbon-components/scss/globals/scss/vendor/@carbon/type/scss/font-family.scss';
+@import 'carbon-components/scss/globals/scss/vendor/@carbon/layout/scss/breakpoint.scss';
+@import 'carbon-components/scss/globals/scss/typography.scss';
+@import 'carbon-components/scss/globals/scss/vars.scss';
```

Comment:
These should have been added to `src/app.scss`

Reply:
@kellychurchill Fixed.